.releaserc

```diff
@@ -10,15 +10,15 @@
 		"@semantic-release/release-notes-generator",
 		[
 			"@semantic-release/exec", {
-				"prepareCmd": 'sed -ir "s/[0-9]*\.[0-9]*\.[0-9]*/${nextRelease.version}/" erpnext/__init__.py'
+				"prepareCmd": 'sed -ir "s/[0-9]*\.[0-9]*\.[0-9]*/${nextRelease.version}/" pyproject.toml'
```

Comment:
@blaggacao only changing the target file here is not enough. This command will replace _all_ version numbers found in `pyproject.toml`, dependencies included, with the `nextRelease.version`.